src/jit/lsra.h

```diff
@@ -847,11 +847,17 @@ class LinearScan : public LinearScanInterface
     };
 
 private:
+    bool enregisterLocalVars()
+    {
+        return localVarIntervals != nullptr;
```

Comment:
Though note that `LinearScan` doesn't use the `m_` prefixes, and I will keep the existing style (which I prefer). Should we choose to change it, I would propose that we do it all or none.